### ISSUE TITLE
Report `content-build-gql` elapsed time metric to DataDog.

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#11632]
## Description
Probably on Staging only.

## Acceptance Criteria
- [ ] Elapsed time figure for the `content-build-gql` test is reported upstream to DataDog.

**Comment by teeshe** [comment]
content build gql duration metrics are captured in the content-release workflow during the content build step. Metrics are reporting to Datadog [dsva_vagov.content_build.build.gql.duration](https://vagov.ddog-gov.com/metric/explorer?start=1700228203734&end=1700231803734&paused=false#N4Ig7glgJg5gpgFxALlAGwIYE8D2BXJVEADxQEYAaELcqyKBAC1pEbghkcLIF8qo4AMwgA7CAgg4RKUAiwAHOChASAtnADOcAE4RNIKtrgBHPJoQaUAbVBGN8qVoD6gnNtUZCKiOq279VKY6epbINiAiGOrKQdpYZAYgUJ4YThr42gDGSsgg6gi6mZaBZnHKGABuMMhQGhWp9TA4FQB0mVIIcCIITgBGeBBoUC39g8MwxmgtUHjanpIiwABUPCA8ALpUru54mKHh26q7GDGl8WubIBpyaDmg8hi3CJ3KUDgwTpl7GhCZiWiiOBOOSKZTpAFQf6Apz0JjKERuDxoNb8CD2TBYYEKHIgAEiJQbHh8K7yAEIADCUmEMBQIl2aB4QA)
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNjdtQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a800073ad7ad800b3ffdf61862d0fb5140064c60/image.png)

**Comment by ndouglas** [comment]
@BerniXiongA6 **TL;DR**: Just discussed this with Tolase and think it might be better to close this ticket. Maybe treat it as a discovery ticket or something like that, since Tolase did some investigation. I would like his time to be accounted for, since he did some research/learning in this area.

Basically, the idea behind this ticket was to record the time that it takes to run the GraphQL queries and report it to Datadog and have that data for every commit in the CMS codebase. Since it was happening on staging, we would have a consistent environment and set of circumstances for comparing different runs and could see when a performance regression was introduced.

However, this was before we parallelized the tests. When we parallelized tests, we introduced a bunch of random variables that probably defeats the intended use of this work. It didn't really occur to me until I got deep into explaining the rationale of this ticket to Tolase 😩
